src/Components/Server/src/Circuits/DefaultCircuitFactory.cs

```diff
@@ -80,7 +80,8 @@ internal class DefaultCircuitFactory : CircuitFactory
                 .ToArray();
 
             var circuitHost = new CircuitHost(
-                _circuitIdFactory.CreateCircuitId(),
+                circuitId: Guid.NewGuid().ToString(),
```

Comment:
It's just a guid.

Reply:
I would feel better if we paired the Id and the secret together. You could do CircuitId as a class and have .Secret that gets returned to the client and .Id that is used for logging purposes, you could have ToString() simply return the ID but that way the relationship between the two is clear. An ID is paired with a secret, or viceversa. If you want to name it CircuitSecret but have the ID be part of the secret as its ID.